globals/typography.css

```diff
@@ -42,3 +42,38 @@
   font-size: 1em;
   font-weight: 700;
 }
+
+.link {
+  text-decoration: none;
+  color: var(--color-primary);
+  font-style: italic;
+  background-image: linear-gradient(to top,
+    transparent 11%,
+    color(var(--color-primary) a(30%)) 11%,
+    color(var(--color-primary) a(30%)) 16%,
+    transparent 16%
```

Comment:
You lose out in safari...

![screenshot 2016-01-26 14 56 28](https://cloud.githubusercontent.com/assets/1926464/12583928/fea2b2e6-c43c-11e5-825a-5d65f420d095.png)